srcpkgs/cura/template

```diff
@@ -1,15 +1,16 @@
 # Template file for 'cura'
 pkgname=cura
 version=3.6.0
-revision=1
+revision=2
 noarch=yes
 wrksrc="Cura-${version}"
 build_style=cmake
 configure_args="-DCURA_VERSION=${version}"
 pycompile_module="cura"
 pycompile_dirs="usr/lib/cura/plugins"
 makedepends="Uranium libArcus-devel"
-depends="Uranium cura-engine cura-fdm-materials python3-pyserial python3-zeroconf"
+depends="Uranium cura-engine cura-fdm-materials python3-pyserial python3-zeroconf \
+	libSavitar-python3"
```

Comment:
1 whitespace indent, no need for \

Reply:
Done, 1 whitespace indent means one space right